ui/Screen/NetworkDiagnostics/WaitingRoom.svelte

```diff
@@ -0,0 +1,72 @@
+<script lang="typescript">
+  import { onDestroy } from "svelte";
+  import JSONTree from "svelte-json-tree";
+
+  import StateTable from "./StateTable.svelte";
+  import {
+    waitingRoomEventLog,
+    WaitingRoomTransition,
+  } from "../../src/localPeer";
+
+  let lastEvent: WaitingRoomTransition | null = null;
+  let eventsWithoutTicks: WaitingRoomTransition[] = [];
+  const unsub = waitingRoomEventLog.subscribe(e => {
+    lastEvent = e[e.length - 1];
+    eventsWithoutTicks = e.filter(e => e.event.type !== "tick");
+  });
+  onDestroy(unsub);
```

Comment:
Since we’re dealing with svelte stores we can write this more idiomatically and eliminate the `onDestroy`.
```suggestion
  $: lastEvent = $waitingRoomEventLog.slice(-1)[0] || undefined;
  $: eventsWithoutTicks = $waitingRoomEventLog.filter(
    e => e.event.type !== "tick"
  );
```